### PR TITLE
Feature 2

### PR DESCRIPTION
**Pull request by shadrxcc**
The feature-2 branch includes the three column sections of the page; the featured stories, top stories and the debate sections. 
These sections are designed to be responsive on different screen sizes such as mobile devices, tablet devices and desktop devices . Bootstrap grid system was used to achieve this layout, the contents in these columns inclusive. The order property was also used to achieve the same responsiveness on mobile view like the original page. We're open to feedback on how to make this page better!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -103,10 +103,731 @@
...
+			<img class="card-img-top" src="images/americas-best-nursing-homes-2020.png" alt="Card image cap">
+			<div class="position-absolute bottom-0 Sp-insight p-1 start-0 w-15">
+				 RANKINGS
+					</div>
+		</div>
+		<div class="card-body"><span class="redunder fw-bolder text-dark">America's Best Nursing Homes 2021 </span>
+		<span class="small font-roboto d-block text-muted fw-lighter"> In partnership with Statista, we established a ranking providing information and insights to help you during this challenging time. </span>
+	</div>
+	</div>
+</div></div>
```
Comment (soieducation) [inline index.html]: - [x] This section should be centered in the tablet view, check original page for reference

**Review by soieducation — CHANGES_REQUESTED** [review]
# Status: CHANGES REQUIRED :warning: :construction: 
### Hi @shadrxcc, @Ademola101  and @YinkTech 👋. This is a very awesome start. You are on a very good track to make your project look perfect.

### That being said, you have a couple of things to work on before you continue to the next step. Please have a look at the corrections below.

- [x] The header in 320px isn't aligned properly.
- [x] The alignment of the second navbar in desktop view has changed again.
- [ ] In the mobile and tablet view, there's a margin left to the main body which is affecting your header. 
___
**If you are stuck at any point or have any questions, let us know by leaving a comment here. We will be happy to help**
HAPPY CODING :computer:

File: index.html (hunk 1 above)
Comment (Ademola101) [reply index.html]: Thanks for the suggestion

**Review by soieducation — CHANGES_REQUESTED** [review]
# Status: CHANGES REQUIRED :warning: :construction: 
### Hi @shadrxcc, @Ademola101  and @YinkTech 👋. Good work on implementing the changes required from the last review.

### However, one of the required changes is yet to be resolved. Resolve this to get this milestone approved.

- [ ] In the mobile and tablet view, there's a margin-left to the main body which is affecting your header.
___
**If you are stuck at any point or have any questions, let us know by leaving a comment here. We will be happy to help**
HAPPY CODING :computer:

**Review by soieducation — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @shadrxcc, @Ademola101  and @YinkTech :wave: 

### You have done an amazing job on this milestone, congratulations! Your project is approved :clap: :clap: :clap: 

### Good luck with your next milestones :rocket: 

### Remember to submit your project completion form individually
___
**If you are stuck at any point or have any questions, let us know by leaving a comment here. We will be happy to help**
HAPPY CODING :computer:

**Comment by Ademola101** [comment]
Thanks for the code review and suggestions. It has been very helpful

**Comment by shadrxcc** [comment]
thank you for the code review and suggestions.

**Pull request merged**